README.md

```diff
@@ -118,11 +129,41 @@ from dataverse.etl import ETLPipeline
 etl_pipeline = ETLPipeline()
 spark, dataset = etl_pipeline.run(config=ETL_config, verbose=True)
 ```
+ETLPipeline is an object designed to manage the ETL processes. By inserting `ETL_config` which is defined in the previous step into ETLpipeline object and calling the `run` method, stacked ETL blocks are sequentially executed.
+
 ```python
 # 3. Result file is saved on the save_path
 ```
+As the example gave `save_path` argument to the last block of `ETL_config`, data passed through the process will be saved on the given path.
+
 <br>
 
+## 🌌 Modules
+Currently, a total of 43 functions are registered as the ETL process, which means they are available for use.
```

Comment:
Is it right that the number of functions can be changed ?
If so, I think it would be better not to appear the specific number of functions in this README.

Reply:
i agree. those would be better placed elsewhere, such as in a GitBook or API references.